pkg/backends/ibmsecretmanager.go

```diff
@@ -72,6 +72,10 @@ func (i *IBMSecretManager) GetSecrets(path, _ string) (map[string]interface{}, e
 			return nil, err
 		}
 
+		if secret == nil || len(secret.Data) == 0 {
```

Comment:
Is there a test for this scenario? Seems it'd be good to have

Reply:
Yes, https://github.com/IBM/argocd-vault-plugin/pull/75/files#diff-53be4adc6f3b43667b953de1f666e083e6f74ffb6065db8715b0406e84e97f64R69 i have an id of a secret that doesn't exist

Reply:
Great, I should've looked at the whole file